docs/JAX_visualizing_models_metrics.md

```diff
@@ -0,0 +1,238 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.15.2
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# JAX and Tensorboard / NNX Display
```

Comment:
In the beginning or in the end it could be good to have some tb API links to read.

Reply:
Updated and included - at the end we have official TB getting-started and the readthedocs for Treescope